cairo/ethereum_rlp/rlp.cairo

```diff
@@ -242,9 +240,56 @@ func Extended__eq__(left: Extended, right: Extended) -> bool {
         return res;
     }
 
-    with_attr error_message("Unimplemented for types other than bytes and sequence") {
-        jmp raise.raise_label;
+    // Bytearray case
+    if (left.value.bytearray.value != 0) {
+        if (right.value.bytearray.value != 0) {
+            let res = Bytes__eq__(left.value.bytearray, right.value.bytearray);
+            return res;
+        }
+        let res = bool(0);
+        return res;
```

Comment:
```suggestion
    if (left.value.bytearray.value != 0 and right.value.bytearray.value != 0) {
          let res = Bytes__eq__(left.value.bytearray, right.value.bytearray);
          return res;
        }
```

Reply:
not sure this is equivalent because we want to early return false otherwise. the ideal would be

if (A and B){
  return X
} else {
  return Y
}

but cairo doesn't support this